docs/source/write_first_app.rst

```diff
@@ -43,6 +43,21 @@ After completing this tutorial you should have a basic understanding of how Fabr
 applications and smart contracts work together to manage data on the distributed
 ledger of a blockchain network.
 
+Before you begin
+----------------
+
+In addition to the standard :doc:`prereqs` for Fabric, this tutorial leverages the Hyperledger Fabric SDK for Node.js. Node.js version 8 is supported from 8.9.4 and higher. Node.js version 10 is supported from 10.15.3 and higher. Before attempting this tutorial, you should install Node.js if it is not already installed. Unsure what version you have installed? Run `node -v` to check. 
+
+  - `Node.js <https://nodejs.org/en/download/>`__ download
+
+.. note:: Installing Node.js will also install NPM, however it is recommended
+          that you confirm the version of NPM installed. You can upgrade
+          the ``npm`` tool with the following command:
+
+.. code:: bash
+
+  npm install npm@5.6.0 -g
```

Comment:
Again, I think it is Node 10 and 12, and npm 6 now.  Better to point to Node.js SDK docs rather than repeat this information multiple times in Fabric docs.

Reply:
Changed this entire section to:
In addition to the standard :doc:`prereqs` for Fabric, this tutorial leverages the Hyperledger Fabric SDK for Node.js. See the Node.js SDK `README <https://github.com/hyperledger/fabric-sdk-node#build-and-test>`__ for a up to date list of prerequisites.